src/css/components/mobile-tablet-menu.css

```diff
@@ -0,0 +1,128 @@
+/* Initial hidden state for overlay and menu */
+.menu-overlay {
+  visibility: hidden;
+  opacity: 0;
+  transition: opacity 0.5s ease, visibility 0.5s ease;
+}
+
+/* Shown state with is-shown class */
+.menu-overlay.is-shown {
+  visibility: visible;
+  opacity: 1;
+}
+
+/* Menu Overlay */
+.menu-overlay {
+  position: fixed;
+  top: 0;
+  right: 0;
+  width: 100%;
+  height: 100%;
+  background: rgba(18, 20, 23, 0.4); /* #12141766 */
+  z-index: 999;
+  opacity: 1;
+}
+
+/* Menu Container */
+.menu {
+  position: fixed;
+  top: 0;
+  right: 0;
+  background: #4dc274;
+  width: 240px;
+  height: 100vh;
+  z-index: 1000;
+}
+
+/* Close Button */
+.menu-close {
+  position: absolute;
+  top: 30px;
+  right: 20px;
+  width: 32px;
+  height: 32px;
+  cursor: pointer;
+}
```

Comment:
<img width="229" alt="image" src="https://github.com/user-attachments/assets/dfcd69df-df1b-4c51-8ee4-6a1a02852110" />
Instead use absolute for menu-close, very high margin top for content and absolute for socials we can use functional of flex.

flex-direction: column;
and use stretch for container with 3 blocks.